wpilibj/src/main/java/edu/wpi/first/wpilibj/event/BooleanEvent.java

```diff
@@ -167,7 +171,29 @@ public boolean getAsBoolean() {
    * Creates a new debounced event from this event - it will become active when this event has been
    * active for longer than the specified period.
    *
-   * @param seconds The debounce period.
+   * @param period The debounce period.
+   * @return The debounced event (rising edges debounced only).
+   */
+  public BooleanEvent debounce(Time period) {
+    return debounce(period.in(Seconds));
+  }
+
+  /**
+   * Creates a new debounced event from this event - it will become active when this event has been
+   * active for longer than the specified frequency.
+   *
+   * @param frequency The debounce frequency.
+   * @return The debounced event (rising edges debounced only).
+   */
+  public BooleanEvent debounce(Frequency frequency) {
```

Comment:
Does it really make sense to debounce with a given frequency? I suppose maybe there's some weird connection with a high pass filter, but since is discrete it doesn't really make sense.
The doc comment is also weird- "longer than the specified frequency" doesn't really make sense.